src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1240 @@
+        'highFrequency': 3695000000
+    }
+
+    # Update the location 'X' of CBSD devices to be near FSS sites
+    # 'KA413' at Albright,WV
+    device_a['installationParam']['latitude'] = 39.57327
+    device_a['installationParam']['longitude'] = -79.61903
+
+    # 'E000306' at Andover,ME
+    device_b['installationParam']['latitude'] = 44.63367
+    device_b['installationParam']['longitude'] = -70.69758
+
+    # Creating conditionals for Cat B devices
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+    del device_b['measCapability']
+
+    conditionals = [conditionals_b]
+
+    # Create the actual config.
+    config = {
+        'registrationRequest': [device_a, device_b],
+        'grantRequests': [
+            [grant_g1_a, grant_g1_b],
+            [grant_g2_a, grant_g2_b]],
+        'expectedResponseCodes': [(400,), (0,)],
+        'conditionalRegistrationData': conditionals
+        # TODO
+        # Need to add data base configurations
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_FDB_6_default_config)
+  def test_WINNF_FT_S_FDB_6(self, config_filename):
+    """GWBL Database Update: GWBL Modification."""
+
+    # Load the configuration file
+    config = loadConfig(config_filename)
+    # Very light checking of the config file.
+    self.assertEqual(len(config['grantRequests']),
+                     len(config['expectedResponseCodes']))
+
+    # Step 1: Register device(s) 'C'(at location 'X') with conditional registration data
+    cbsd_ids = self.assertRegistered(config['registrationRequest'],
+                                     config['conditionalRegistrationData'])
+
+    # TODO
+    # Step 2: Create FSS database which includes at least one FSS site near
+    # CBSD location 'X'.
+
+    # TODO
+    # Step 3: Create GWBL database which includes at least one GWBL site near
+    # CBSD location 'X'.
+
+    # Step 4: Trigger daily activities
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 5: Send Grant Request 'G1'.
+    # Forming of 'G1' grants for all the configured grants
+    addCbsdIdsToRequests(cbsd_ids, config['grantRequests'][0])
+    grant_request_g1 = {'grantRequest': config['grantRequests'][0]}
+
+    grant_response_g1 = self._sas.Grant(grant_request_g1)['grantResponse']
+
+    # Check the grant response code is as expected (SUCCESS or INTERFERENCE)
+    grant_ids = []
+    for resp in grant_response_g1:
+      self.assertIn(resp['response']['responseCode'], config['expectedResponseCodes'][0])
```

Comment:
Why `[0]`?

Reply:
FDB.6     Expected response codes for 'G1' grant will be checked.